src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
Yes, the call localization is more like an "internal" localization which tries to regroup inputs for a particular call in one directory, which in the end is just moving a file a few directories up and down.
Let's say you define only "copy" in the conf file as a localization strategy, if we were to use that also for call localization, the same file in the workflow inputs would be copied for every call that refers to it which is sub-optimal and unnecessary in my opinion.
The configuration localization strategies are still used to localize in workflow options, but I think it's worth trying to link for the second level of localization even if it's not part of the conf.